src/operations_client.ts

```diff
@@ -428,9 +428,9 @@ OperationsClient.prototype.deleteOperation = function(
 };
 
 function OperationsClientBuilder(gaxGrpc) {
-  if (!(this instanceof OperationsClientBuilder)) {
-    return new OperationsClientBuilder(gaxGrpc);
-  }
+  // if (!(this instanceof OperationsClientBuilder)) {
+  //   return new OperationsClientBuilder(gaxGrpc);
+  // }
```

Comment:
@alexander-fenster @stephenplusplus another one of those instances where we have to make a call on who to break.  Ideas on how this is used most commonly in the wild?